backend/routes/api.php

```diff
@@ -50,6 +50,8 @@
             Route::get('/new', 'VisitorController@getNewVisitors');
             Route::get('/new/count', 'VisitorController@getNewVisitorsCountForFilterData');
             Route::get('/bounce-rate/total', 'VisitorController@getBounceRate');
+            Route::get('/bounce-rate/total', 'VisitorController@getBounceRate');
```

Comment:
why did you duplicate route? be attentive

Reply:
Fixed